### DISCUSSION TITLE
Hardhat Block latency. (During mainnet fork)

### DISCUSSION THREAD

**Discussion by snowkidind** [NomicFoundation/hardhat#4820] [General]
I should be able to override the 6 block minimum block latency. Its my system there is no reason this should require 6 blocks. If something ends up getting orphaned that's my problem not yours.

```
You are forking from block 19155665, which has less than 6 confirmations, and will affect Hardhat Network's performance.
Please use block number 19155664 or wait for the block to get 1 more confirmations.

```

**Comment by fvictorio** [comment]
But this is just a warning, isn't it? Nothing should be preventing you from doing it if you want.
